vm/src/stdlib/socket.rs

```diff
@@ -449,6 +449,28 @@ fn socket_htonl(host: u32, vm: &VirtualMachine) -> PyResult {
     Ok(vm.new_int(host.to_be()))
 }
 
+fn socket_htons(host: u16, vm: &VirtualMachine) -> PyResult {
+    Ok(vm.new_int(host.to_be()))
+}
+
+fn socket_ntohl(network: u32, vm: &VirtualMachine) -> PyResult {
+    if cfg!(target_endian = "big") {
+        Ok(vm.new_int(network))
+    }
+    else {
+        Ok(vm.new_int(network.to_le()))
+    }
+}
+
+fn socket_ntohs(network: u16, vm: &VirtualMachine) -> PyResult {
+    if cfg!(target_endian = "big") {
+        Ok(vm.new_int(network))
+    }
```

Comment:
These seem like no-ops -- if the machine is big endian, leave it as is, or else if it's little endian, convert to little endian, i.e. leave it as is. You might be looking for `u{16,32}::from_{le,be}`

Reply:
Ah.  Yeah, I couldn't find those before.